test/blockstore-test.js

```diff
@@ -358,32 +359,32 @@ describe('BlockStore', function() {
     describe('filepath', function() {
       it('will give correct path (0)', () => {
         const filepath = store.filepath(types.BLOCK, 0);
-        assert.equal(filepath, '/tmp/.bcoin/blocks/blk00000.dat');
+        assert.equal(filepath, path.join(location, 'blk00000.dat'));
```

Comment:
Okay, this has the unfortunately consequence that it's now comparing against a dynamic expected result that is not defined within the test itself.

Reply:
Hmm, absolute paths are really OS dependent though, we could let it choose between two static paths if we want to drop the path join here but that would also require if/else casing between OS environments.